app/products/calls/utils.ts

```diff
@@ -99,6 +107,23 @@ export function isMultiSessionSupported(callsVersion: CallsVersion) {
     );
 }
 
+export function isHostControlsSupported(callsVersion: CallsVersion) {
+    return isMinimumServerVersion(
+        callsVersion.version,
+        Calls.HostControlsCallsVersion.MAJOR_VERSION,
+        Calls.HostControlsCallsVersion.MIN_VERSION,
+        Calls.HostControlsCallsVersion.PATCH_VERSION,
+    );
+}
+
+export function isHostControlsAllowed(config: CallsConfigState) {
+    return isHostControlsSupported(config.version) &&
+        (config.sku_short_name === license.SKU_SHORT_NAME.E10 ||
+            config.sku_short_name === license.SKU_SHORT_NAME.E20 ||
+            config.sku_short_name === license.SKU_SHORT_NAME.Professional ||
+            config.sku_short_name === license.SKU_SHORT_NAME.Enterprise);
+}
```

Comment:
Still I wish the server could control this (e.g. expose available functionalities through an API endpoint) so that we wouldn't be stuck on what's on the client.

Reply:
Yep, done in [9d6d7b1](https://github.com/mattermost/mattermost-mobile/pull/7905/commits/9d6d7b11cb9d7c9d34b6af135d13a16fc523ac98) and https://github.com/mattermost/mattermost-plugin-calls/pull/709 and https://github.com/mattermost/calls-common/pull/28